lib/bugsnag/notification.rb

```diff
@@ -235,17 +237,13 @@ def deliver
           :exceptions => exception_list,
           :severity => self.severity,
           :groupingHash => self.grouping_hash,
-        }
+          :metaData => @meta_data
+        }.reject {|k,v| v.nil? }
 
         payload_event[:device] = {:hostname => @configuration.hostname} if @configuration.hostname
 
         payload_event = Bugsnag::Helpers.cleanup_obj_encoding(payload_event)
```

Comment:
@lowensk We probably want to run `cleanup_obj` on the meta-data after the user's custom handlers (they might add stupid stuff to the meta-data at that point).